dongkyu/initial-settings-express/myapp/app.js

```diff
@@ -53,6 +53,42 @@ app.post("/users", async (req, res, next) => {
 })
 
 
+app.post("/posts", async (req, res) => {
+  const { title, content, user_id } = req.body
```

Comment:
```suggestion
  const { title, content, userId } = req.body
```
javascript에서는 snake_case가 아닌 camelCase를 사용하도록 해주세요~